homeassistant/components/uptimerobot/config_flow.py

```diff
@@ -44,14 +44,11 @@ async def _validate_input(
 
         try:
             response = await uptime_robot_api.async_get_account_details()
-        except UptimeRobotAuthenticationException as exception:
-            LOGGER.error(exception)
+        except UptimeRobotAuthenticationException:
             errors["base"] = "invalid_api_key"
-        except UptimeRobotException as exception:
-            LOGGER.error(exception)
+        except UptimeRobotException:
             errors["base"] = "cannot_connect"
-        except Exception as exception:  # noqa: BLE001
-            LOGGER.exception(exception)
```

Comment:
We should only log the errors if we don't know what is going on, so it's justified if we log on a bare exception (as something happened that we did not expect) and we want to be able to find that out in more detail)

Reply:
Make sense.
I would vote to have a pre-commit check for those wide exceptions in order to align all integrations to the same approach.